javascript/javascript3/week2/Task1_Movies_Exercise/movies.exercise.js

```diff
@@ -0,0 +1,39 @@
+// TASK 1 - Movies exercise
+// Create an array of bad movies
+
+const url =
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+
+const badMovies = fetch(url)
+  .then((response) => response.json())
+  .then((data) => data.filter((item) => item.rating <= 4 && item.year));
+console.log(badMovies);
+
+// 2. Creat an array of bad movies since year 2000
+
+const getBadMoviesSince2000 = async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data.filter((item) => item.rating <= 4 && item.year >= 2000);
+};
+console.log(getBadMoviesSince2000());
+
+///////////////////////////////////////////////////////////////////////////////
+
+async function getData(url) {
+  let response = await fetch(url);
+  if (response.status >= 200 && response.status < 300) {
+    let data = response.json();
+    return data;
+  } else {
+    throw new Error(response.status);
```

Comment:
Argument of type 'number' is not assignable to parameter of type 'string | undefined'.ts(2345)

Reply:
![DeleteMe1](https://user-images.githubusercontent.com/81259898/133083702-3f7a840f-4cbe-4bb3-b573-7cc9180f7cf9.PNG)

Reply:
IF I run the code on my machine it works without any errors. I have no clue how to fix this problem.
-> this has to be the line that is giving you an error : if (response.status >= 200 && response.status < 300.
Open for suggestions.